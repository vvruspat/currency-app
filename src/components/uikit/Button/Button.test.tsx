import { render, screen } from "@testing-library/react";
import { Button } from ".";

test("renders Button", () => {
  render(<Button data-testid="test-button">Test Button</Button>, {});
  const ButtonElement = screen.getByTestId("test-button");
  expect(ButtonElement).toBeInTheDocument();
  expect(ButtonElement).toHaveTextContent("Test Button");
  expect(ButtonElement).toMatchSnapshot();
});
