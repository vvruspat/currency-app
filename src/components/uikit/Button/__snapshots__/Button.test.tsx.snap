// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button 1`] = `
<button
  class="button primary-button"
  data-testid="test-button"
>
  Test Button
</button>
`;
